Feature/create_logs_test-develop

pparfrey opened the pull request.

pparfrey committed:
TEST: Added test for create_logs_json.py

pparfrey committed:
TEST: Added test file for create_logs_json.py

pparfrey committed:
STYLE: Renamed function

pparfrey committed:
TEST: Changed the way paths are handled

EricWiener commented:
Hey Patrick. Looks good. Some small changes:

### Could you rebase off develop:
```
# Make sure you are on your branch
$ git checkout feature/create_logs_test-develop
$ git rebase origin/develop
```

### Could you create a correct output.json file to compare to
Instead of manually creating the correct row entries, could you please save a correct output.json file with the expected results. You can look at `tests/test_supervisely_3d_to_universal.py` for an example of this.

### Please make sure to run `tox` and that you pass the tests

### Please make sure you pass the linter
Your code has syntax mistakes, so this likely means you didn't setup your environment with commitlint correctly.
Please see the docs on details for how to setup your environment: https://universal-devkit.readthedocs.io/en/latest/contributing.html

pparfrey committed:
TEST: Added test file for create_logs_json.py

EricWiener commented:
Hey Patrick. You still aren't passing any of the tests. Please make sure you are running Tox.

Additionally, your commit messages should have a commit message body. The commit linter catches this for you, so your setup definitely isn't correct.

pparfrey committed:
TEST: Added test file to compare to directly

pparfrey committed:
TEST: Fixed import

pparfrey committed:
FIX: Changed type of logs_dir_path to str and imports

pparfrey committed:
FIX: Fixed how paths are handled

pparfrey committed:
FIX: Fixed imports, unused variables, file removal

EricWiener committed:
FEAT: cleaned up create_logs_json

Added:
- Directory is now created if it doesn't exist
- Reduced redundant code
- No longer re-converting paths to strings
- Test case now converts paths to strings before calling create_logs
